samples/pmic/native/npm13xx_one_button/README.rst

```diff
@@ -1,13 +1,13 @@
-.. _npm1300_one_button:
+.. _npm13xx_one_button:
 
-nPM1300: One button
+nPM13xx: One button
```

Comment:
`nPM13xx` is not allowed as exception on https://nordicsemi.atlassian.net/wiki/spaces/MAR/pages/132940950/Product+names#nPM-Family.
Therefore:
```suggestion
nPM1300 and nPM1304: One button
```